wp-postratings.php

```diff
@@ -92,7 +92,9 @@ function the_ratings($start_tag = 'div', $custom_id = 0, $display = true) {
 	// Check To See Whether User Has Voted
 	$user_voted = check_rated($ratings_id);
 	// HTML Attributes
-	if( is_single() || is_page() ) {
+	$ratings_options = get_option('postratings_options');
+
```

Comment:
Need to add this
```php
$ratings_options['richsnippet'] = isset($ratings_options['richsnippet']) ? $ratings_options['richsnippet'] : 1;
```

Because when user has an old copy of WP-Ratings 'richsnippet' will be undefined